acceptance/tests/fixtures/bases/openshift/network-attachment.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: consul-cni
+spec:
+  config: '{
+            "cniVersion": "0.3.1",
+            "type": "consul-cni",
+            "cni_bin_dir": "/var/lib/cni/bin",
+            "cni_net_dir": "/etc/kubernetes/cni/net.d",
+            "kubeconfig": "ZZZ-consul-cni-kubeconfig",
+            "log_level": "info",
```

Comment:
It won't give very much logging output but please set this to 'debug' instead of 'info'.

Reply:
I set this to debug.